### DISCUSSION TITLE
I can only find one branch(FE test)

### DISCUSSION THREAD

**Discussion by FikaKarin** [DevSkillsHQ/discusssions#12] [Help] [answered]
I'm suppose to work from implement branch but there is only one branch available: main ?

**Comment by fenske** [comment, marked as answer]
Hi @FikaKarin. 👋 I'm Anton from DevSkills. That's by design as the expectation is that you create it.
![CleanShot 2023-03-22 at 11 25 20@2x](https://user-images.githubusercontent.com/1162212/226874833-f767e5f3-e9f6-416e-b816-0efc6a624e28.png)

Hope this helps.
